opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -183,10 +188,14 @@ def to_metric(self, desc, tag_values, agg_data):
             for ii, bound in enumerate(agg_data.bounds):
                 cum_count += agg_data.counts_per_bucket[ii]
                 points[str(bound)] = cum_count
+            # Prometheus requires buckets to be sorted, and +Inf present.
+            # In OpenCensus we don't have +Inf in the bucket bonds so need to
+            # append it here.
+            points["+Inf"] = agg_data.count_data
```

Comment:
What's this bug fix in here? A test demonstrating the problem would be useful.  I thought this code runs through the Prometheus Python client first before being exported? I think it would be good for us to file and diagnose that bug separately than sneak in a bug fix here without figuring out its extent.

Reply:
See https://github.com/prometheus/client_python/blob/master/prometheus_client/metrics_core.py#L204.

I've removed this commit from this PR for now.